src/server/tcp.rs

```diff
@@ -43,7 +43,11 @@ impl Server {
         S::Instance: Send + Sync + 'static,
     {
         let service = Arc::new(service);
-        let listener = TcpListener::bind(self.socket_addr).await?;
+        let socket = Socket::new(Domain::IPV4, Type::STREAM, None)?;
+        socket.set_nodelay(true)?;
+        socket.bind(&self.socket_addr.into())?;
+        socket.listen(1024)?;
```

Comment:
Could you add some notes why `1024` is used? Ideally point to some documentation somewhere.

Reply:
A (private) constant with comments would be appropriate here.

Reply:
The param to listen is the number of connections allowed queued as I read it from [listen(2)](https://man7.org/linux/man-pages/man2/listen.2.html)
I dont think it's the number of allowed simultanious connections, just "not-yet-accepted" connections. 

I have added a const with comments